openapi/src/parts/referenced_terra_workspace.yaml

```diff
@@ -0,0 +1,94 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/referenced/terra/workspaces:
```

Comment:
curious about why we are adding the terra prefix. what workspaces are we trying to distinguish from?

Reply:
True. That doesn't seem necessary in the API. If we were being strict about REST, the object would be a "terraworkspace" not a "terra/workspace", since there isn't a "terra" collection.

I like it (or something!) in the code as a way to distinguish a referenced workspace from "this" workspace.

Reply:
No other workspace. I put this in the "terra" directory/namespace, similar to the azure namespace. There are a bunch of resources under `/api/workspaces/v1/{workspaceId}/resources/controlled/azure`. In the future, there could be other Terra resources under `/api/workspaces/v1/{workspaceId}/resources/referenced/terra`.